src/admin/runners/fixInstantDataTypeRunner.js

```diff
@@ -0,0 +1,232 @@
+const deepEqual = require('fast-deep-equal');
+const { BaseBulkOperationRunner } = require('./baseBulkOperationRunner');
+const { RethrownError } = require('../../utils/rethrownError');
+const { FhirResourceCreator } = require('../../fhir/fhirResourceCreator');
+const { resourcesWithDateTimeFields } = require('../utils/resourcesWithDateTimeFields')
+const {DateColumnHandler} = require("../../preSaveHandlers/handlers/dateColumnHandler");
+class FixInstantDataTypeRunner extends BaseBulkOperationRunner {
+    /**
+     * constructor
+     * @typedef {Object} ConstructorProps
+     * @property {MongoCollectionManager} mongoCollectionManager
+     * @property {MongoDatabaseManager} mongoDatabaseManager
+     * @property {string[]} collections
+     * @property {number} batchSize
+     * @property {AdminLogger} adminLogger
+     * @property {string|undefined} startFromCollection
+     * @property {number|undefined} limit
+     * @property {boolean|undefined} useTransaction
+     * @property {number|undefined} skip
+     * @property {string|undefined} startFromId
+     *
+     * @param {ConstructorProps}
+     */
+    constructor ({
+        mongoCollectionManager,
+        mongoDatabaseManager,
+        collections,
+        batchSize,
+        adminLogger,
+        startFromCollection,
+        limit,
+        useTransaction,
+        skip,
+        startFromId
+    }) {
+        super({
+            mongoCollectionManager,
+            batchSize,
+            adminLogger,
+            mongoDatabaseManager
+        });
+        /**
+         * @type {string[]|undefined}
+         */
+        this.collections = collections;
+
+        /**
+         * @type {string|undefined}
+         */
+        this.startFromCollection = startFromCollection;
+
+        /**
+         * @type {number|undefined}
+         */
+        this.limit = limit;
+
+        /**
+         * @type {boolean|undefined}
+         */
+        this.useTransaction = useTransaction;
+
+        /**
+         * @type {number|undefined}
+         */
+        this.skip = skip;
+
+        /**
+         * @type {string|undefined}
+         */
+        this.startFromId = startFromId;
+    }
+
+    /**
+     * gets all collection names
+     * @returns {Promise<string[]>}
+     */
+    async getAllCollectionNamesAsync () {
```

Comment:
`getAllCollectionNamesAsync` function already exists in `BaseScriptRunner`, please check if we can use that

Reply:
Switched to base method